.travis.yml

```diff
@@ -71,13 +74,15 @@ jobs:
         BITCOIN_CONFIG="--enable-reduce-exports --disable-gui-tests"
 # Win64
     - stage: test
+      name: 'Win64'
       env: >-
         HOST=x86_64-w64-mingw32
         PACKAGES="python3 nsis g++-mingw-w64-x86-64 wine-binfmt wine64"
         GOAL="deploy"
         BITCOIN_CONFIG="--enable-reduce-exports --disable-gui-tests"
 # 32-bit + dash
     - stage: test
+      name: '32-bit + dash'
```

Comment:
I for one value very much to be able to see the build environment variables when looking at the travis overview page for the build. This is lost now as one sees only this identifier. I like to see the environment variables to understand whether the particular build runs functional tests and/or unit tests or not. Maybe this information should be added to the name.

Reply:
I agree! I was sad to see that Travis removed the environment variables from the UI when you add a name. You can see the environment variables still if you go into a build job. Nonetheless, I can certainly add a test type to the job name if we would like.